packages/chain-adapters/src/ethereum/EthereumChainAdapter.test.ts

```diff
@@ -6,6 +6,10 @@
  */
 import * as ethereum from './EthereumChainAdapter'
 
+const EOA_ADDRESS = '0x05a1ff0a32bc24265bcb39499d0c5d9a6cb2011c'
+const CONTRACT_ADDRESS = '0xc770eefad204b5180df6a14ee197d99d808ee52d'
+const ENS_NAME = 'willywonka.eth'
```

Comment:
The owner of this address has publicly associated his identity with this address and ENS name, but I think we should still try and use another ENS/address combination here if possible. How about vitalik.eth instead?

Reply:
That's probably better yeah! I'll use `vitalik.eth` / `0xd8da6bf26964af9d7eed9e03e53415d37aa96045`